package.json

```diff
@@ -75,5 +77,8 @@
   "engines": {
     "node": ">=18",
     "npm": ">=7"
+  },
+  "dependencies": {
```

Comment:
Do we still need this after Amir removed the respective code?

Reply:
You know, good point, I hadn't considered that.  I thought that was a separate dependency but will check.